src/flavors/defaultflavor/config.json

```diff
@@ -419,73 +419,54 @@
     "is_visible_default": true,
     "component": "FeaturedPlacesNavigator"
   },
-  "left_sidebar": {
-    "is_enabled": true,
-    "is_visible_default": false,
-    "panels": [
+  "leftSidebar": {
```

Comment:
`leftSidebar` changes here are unrelated to this PR but were necessary to run the default flavor.